### DISCUSSION TITLE
Camera positions showing up as "ghosts" in renders

### DISCUSSION THREAD

**Discussion by azadbal** [NVlabs/instant-ngp#322] [Q&A]
I'm seeing dark "ghosting" of the camera positions in the render. 

Here's what I mean:
https://gfycat.com/glassleftcomet
https://gfycat.com/flimsyelderlygalapagosmockingbird

Limbike Dataset: https://drive.google.com/drive/folders/12I2pT4JGWlbW-JVzXrO_4y2VdWUjo7cJ?usp=sharing

Camera: Sony A7rii 15mm voigtlander
RAW processing: DxO Photolab - no distortion correction applied, only camera downscale to 2 megapixels

**Comment by Tom94** [comment]
Hi, you might be able to reduce the ghosting by decreasing the near distance in the NeRF rendering options like so:
![image](https://user-images.githubusercontent.com/4923655/157392372-e4bb57d2-9504-4f47-94f9-e64457334d4d.png)

If this doesn't help, there's a high likelihood that there's a fundamental discrepancy among the images learned by the cameras (e.g. automatic exposure during the capture), which can't be incorporated into the 3D model and thus manifests as floaters. You could try ticking these three boxes after the first ~1-2k iterations:
![image](https://user-images.githubusercontent.com/4923655/157392566-551ca65a-f30e-4ec0-8c16-3bfb6d314d27.png)

**Comment by azadbal** [comment]
Hi @Tom94, thanks for replying!

Decreasing the near distance variable actually increased the visibility of the camera positions for me. I know both the exposures, focal length, and focus were consistent throughout the datasets (cause I was capturing these for photogrammetry scanning).

Funnily enough, when retraining with the default params, the ghosting now seems way less prominent than in the examples I provided in my post. Not sure if anything changed or if there's just some inherent indeterministic properties when training the same dataset with the same settings.
